tests/fe/fe_enriched_color_07.cc

```diff
@@ -0,0 +1,2038 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 - 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+/*
+ * Testing the ColorEnriched::Helper class by solving problems
+ */
+
+#include "../tests.h"
+#include <deal.II/base/utilities.h>
+#include <deal.II/base/function.h>
+#include <deal.II/base/exceptions.h>
+#include <deal.II/base/parsed_function.h>
+#include <deal.II/base/conditional_ostream.h>
+#include <deal.II/base/function_cspline.h>
```

Comment:
Regarding error - tests/fe/fe_enriched_color_07.cc

I already had the header for cspline. The error doesn't show up in my runs!